addons/live-featured-project/addon.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/master/1/1.2.json",
+  "name": "Live featured project",
+  "description": "Shows interactive embed of the featured project, directly from the user's profile page. Based on the implementation on Mega Scratch Userscript.",
+  "credits": [
+    {
+      "name": "GRA0007 (original)",
+      "link": "https://github.com/MegaScratchUserscript/Mega-Scratch-Userscript/blob/master/originals/GRA0007/scratchultimate.user.js#L23-L46"
+    },
+    {
+      "name": "Hans5958 (remake)"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "script.js",
+      "matches": ["https://scratch.mit.edu/users/*"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/users/*"]
+    }
+  ],
+  "settings": [
+    {
+      "id": "showMenu",
+      "name": "Show project menu (top bar)",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "id": "autoPlay",
+      "name": "Auto-play projects",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "id": "forceAlternative",
+      "name": "Force using alternative player",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "id": "alternativePlayer",
```

Comment:
Instead of two settings for the player - can you make it a single select, with values: default (Scratch vanilla), TurboWarp, Forkphorus? That would make more sense imo. We're trying not to have selects that only work when a checkbox is enabled, for clarity, by instead adding a "none" option in the select.

Reply:
I don't think so. The way it works that it loads Scratch's embed first, then use the alternative player if it fails. This is how the implementation on MSU works, and the intended way. The option to use the alternative player is added just added just in case.